Fails when building micro and embed at the same time

Use command: `bin/spc build mbstring,openssl --build-all --debug`

```
make: *** No rule to make target `install-micro', needed by `install-binaries'.  Stop.
make: *** Waiting for unfinished jobs....
Installing PHP SAPI module:       embed
Installing PHP FPM binary:        /Users/jerry/project/git-project/static-php-cli/buildroot/sbin/
Installing PHP CLI binary:        /Users/jerry/project/git-project/static-php-cli/buildroot/bin/
Installing PHP CLI man page:      /Users/jerry/project/git-project/static-php-cli/buildroot/php/man/man1/
cp: libs/libphp.a: No such file or directory
make: *** [install-sapi] Error 1
Installing PHP FPM defconfig:     /Users/jerry/project/git-project/static-php-cli/buildroot/etc/
Installing PHP FPM man page:      /Users/jerry/project/git-project/static-php-cli/buildroot/php/man/man8/
Installing PHP FPM status page:   /Users/jerry/project/git-project/static-php-cli/buildroot/php/php/fpm/


```

--- a/src/SPC/builder/linux/LinuxBuilder.php
+++ b/src/SPC/builder/linux/LinuxBuilder.php
@@ -10,6 +10,7 @@ use SPC\builder\traits\UnixBuilderTrait;
 use SPC\exception\FileSystemException;
 use SPC\exception\RuntimeException;
 use SPC\exception\WrongUsageException;
+use SPC\store\FileSystem;
 use SPC\store\SourcePatcher;
 
 class LinuxBuilder extends BuilderBase
@@ -223,6 +224,9 @@ class LinuxBuilder extends BuilderBase
         }
         if ($enableEmbed) {
             logger()->info('building embed');
+            if ($enableMicro) {
+                FileSystem::replaceFileStr(SOURCE_PATH . '/php-src/Makefile', 'OVERALL_TARGET =', 'OVERALL_TARGET = libphp.la');
+            }
             $this->buildEmbed($extra_libs, $use_lld);
         }
 

--- a/src/SPC/builder/macos/MacOSBuilder.php
+++ b/src/SPC/builder/macos/MacOSBuilder.php
@@ -10,6 +10,7 @@ use SPC\builder\traits\UnixBuilderTrait;
 use SPC\exception\FileSystemException;
 use SPC\exception\RuntimeException;
 use SPC\exception\WrongUsageException;
+use SPC\store\FileSystem;
 use SPC\store\SourcePatcher;
 
 class MacOSBuilder extends BuilderBase
@@ -195,6 +196,9 @@ class MacOSBuilder extends BuilderBase
         }
         if ($enableEmbed) {
             logger()->info('building embed');
+            if ($enableMicro) {
+                FileSystem::replaceFileStr(SOURCE_PATH . '/php-src/Makefile', 'OVERALL_TARGET =', 'OVERALL_TARGET = libphp.la');
+            }
             $this->buildEmbed();
         }
 

--- a/src/SPC/store/SourcePatcher.php
+++ b/src/SPC/store/SourcePatcher.php
@@ -141,6 +141,8 @@ class SourcePatcher
         }
         FileSystem::replaceFileRegex(SOURCE_PATH . '/php-src/main/php_config.h', '/^#define HAVE_OPENPTY 1$/m', '');
 
+        FileSystem::replaceFileStr(SOURCE_PATH . '/php-src/Makefile', 'install-micro', '');
+
         // call extension patch before make
         foreach ($builder->getExts() as $ext) {
             if ($ext->patchBeforeMake() === true) {

